task.go

```diff
@@ -55,6 +56,7 @@ type CheckpointTaskOpts func(*CheckpointTaskInfo) error
 
 type TaskInfo struct {
 	Checkpoint *types.Descriptor
+	RootMounts []mount.Mount
```

Comment:
Should we rename `CreateTaskRequest.RootFS` to `RootFSMounts`?

@AkihiroSuda How does this fit with the rootfs specification?